KTLOSNaxxLoot.json

```diff
@@ -613,17 +613,17 @@
 	},
 	"Kel'Thuzad":{
 		"The Phylactery of Kel'Thuzad":{
-			"prio":["Killstep(sunnyD)","ThePrestige(crush)","Johnscar(sunnyD)","LC","EP/GP"],
+			"prio":["Johnscar(sunnyD)","Killstep(sunnyD)","LC","EP/GP"],
```

Comment:
wait why is Prestige being removed from Phylactery

Reply:
I thought he got it but apparently he passed?